### ISSUE TITLE
Suggestion: outsource `tsconfig.json` loading

### ISSUE THREAD

**Issue by OliverJAsh** [dividab/tsconfig-paths#98]
In the past we've [ran into bugs around this](https://github.com/dividab/tsconfig-paths/pull/95/files), so it might be better to outsource it.

E.g. https://github.com/TypeStrong/tsconfig

**Comment by jonaskello** [comment]
IIRC we actually used that package in the very beginning but then we had feature requests for inheritance and comments which, at least at that time the tsconfig package did not support. So we moved to parsing with a custom JSON package to support comments and added our own inheritance functionality.

**Comment by OliverJAsh** [comment]
You're right—that package still does not support config inheritance (`extends`). Perhaps we should extract the code that's already in this library into a separate repository, so others can benefit from our work. 😄

**Comment by jonaskello** [comment]
Yes that sounds like a good idea :-). Perhaps the tsconfig package is accepting PRs so we can add the features to that package?

**Comment by blakeembrey** [comment]
@OliverJAsh @jonaskello If either of you want to maintain that package, let me know! Happy to add you as owners.

**Comment by chanceaclark** [comment]
If y'all update TypeScript support to 3.2+ you can use `tsc --showConfig` to automatically generate the entire config.
https://www.typescriptlang.org/docs/handbook/release-notes/overview.html#the-new---showconfig-flag
